orc8r/cloud/docker/docker-compose.yml

```diff
@@ -60,174 +21,23 @@ services:
       - fluentd
     command: ["/bin/sh", "-c", "/usr/local/bin/wait-for-it.sh -s -t 30 postgres:5432 && /usr/bin/supervisord"]
     restart: always
-
-  orchestrator:
-    <<: *orc8r_service
-    build:
-      context: /tmp/magma_orc8r_build
-      dockerfile: $PWD/Dockerfile
-    depends_on:
-      # Run-once
-      - dev_setup
-      # Orc8r services
-      - accessd
-      - bootstrapper
-      - certifier
-      - configurator
-      - device
-      - directoryd
-      - dispatcher
-      - metricsd
-      - obsidian
-      - service_registry
-      - state
-      - streamer
-      - tenants
-      # Non-core essentials
-      - fluentd
-      - postgres
-    command:
-      - /bin/sh
-      - -c
-      - >
-        /usr/local/bin/wait-for-it.sh -s -t 30 postgres:5432 &&
-        /usr/bin/envdir /var/opt/magma/envdir /var/opt/magma/bin/orchestrator -run_echo_server=true -logtostderr=true -v=0
-
-  accessd:
-    <<: *orc8r_service
-    command:
-      - /bin/sh
-      - -c
-      - >
-        /usr/local/bin/wait-for-it.sh -s -t 30 postgres:5432 &&
-        /usr/bin/envdir /var/opt/magma/envdir /var/opt/magma/bin/accessd -logtostderr=true -v=0
-
-  bootstrapper:
-    <<: *orc8r_service
-    command:
-      - /bin/sh
-      - -c
-      - >
-        /usr/local/bin/wait-for-it.sh -s -t 30 postgres:5432 &&
-        /usr/bin/envdir /var/opt/magma/envdir /var/opt/magma/bin/bootstrapper -cak=/var/opt/magma/certs/bootstrapper.key -logtostderr=true -v=0
-
-  configurator:
-    <<: *orc8r_service
-    command:
-      - /bin/sh
-      - -c
-      - >
-        /usr/local/bin/wait-for-it.sh -s -t 30 postgres:5432 &&
-        /usr/bin/envdir /var/opt/magma/envdir /var/opt/magma/bin/configurator -logtostderr=true -v=0
-
-  ctraced:
-    <<: *orc8r_service
-    command:
-      - /bin/sh
-      - -c
-      - >
-        /usr/local/bin/wait-for-it.sh -s -t 30 postgres:5432 &&
-        /usr/bin/envdir /var/opt/magma/envdir /var/opt/magma/bin/ctraced -logtostderr=true -v=0
-
-  dev_setup:
```

Comment:
Does the supervisord conf have this already?

Reply:
Yes all these items were copied from supervisord conf